src/components/ApplicationShell.jsx

```diff
@@ -55,13 +60,14 @@ export const ApplicationShell = () => {
               <Burger
                 opened={isOpened}
                 onClick={toggleBurger}
-                size="sm"
-                color={theme.colors.gray[6]}
+                size="18px"
```

Comment:
Why did you change value?
"sm" here produce the same result as 18px ("sm" == 1.125rem == 18px).

I suggest not to use hardcoded values, even if result will be the same.

Reply:
of course you are right

Reply:
It's not resolved yet 🙂